### PR TITLE
(#6542) - add install, samples and out directories to .gitignore

### PR DESCRIPTION
**Pull request by su225**
Output directories and directories created for the sake of running tests must be ignored. So add these to .gitignore directory.

### CODE REVIEW

**Comment by ericvn** [comment]
While I agree `out` should maybe had been added with the build tooling update, the install and samples are part of moving tests that I'm wondering what the status will be (see https://github.com/istio/istio.io/pull/6536).

**Comment by su225** [comment]
Sure. Let me wait.

File: .gitignore [hunk 1]
```diff
@@ -13,3 +13,6 @@ public
 resources
 node_modules
 generated
+install
+samples
+out
```
Comment (ericvn) [inline .gitignore]: Can you add a comment: # Contains the built artifacts

File: .gitignore [hunk 2]
```diff
@@ -13,3 +13,6 @@ public
 resources
 node_modules
 generated
+install
```
Comment (ericvn) [inline .gitignore]: Add a comment: # Install generated files

File: .gitignore (hunk 1 above)
Comment (ericvn) [inline .gitignore]: Does this need to be out/?

File: .gitignore (hunk 2 above)
Comment (ericvn) [reply .gitignore]: Based on istio/istio, does this need to be install/** if we are only adding one line?

File: .gitignore (hunk 1 above)
Comment (su225) [reply .gitignore]: Changed.

File: .gitignore (hunk 2 above)
Comment (su225) [reply .gitignore]: Done.

File: .gitignore (hunk 1 above)
Comment (su225) [reply .gitignore]: Done.

**Pull request merged**